### PR TITLE
Catch error in gofmt

### PR DESCRIPTION
**Pull request by cniackz**
Our test is not catching error in `gofmt` in version of go 1.19, look -d is not exiting with status of 1 hence we just print the difference but we don't notice this is actually failing:

https://github.com/golang/go/issues/46289

What I am doing in this PR is to actually catch the issue and fail the test if gofmt is having any difference.

### Dependencies:

* https://github.com/minio/console/pull/2278

### Related: 

* https://github.com/minio/console/pull/2283

### CODE REVIEW

File: verify-gofmt.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+for VARIABLE in pkg cmd cluster restapi
+do
+	DIFF=$(GO111MODULE=on gofmt -d $VARIABLE/)
```
Comment (reivaj05) [inline verify-gofmt.sh:5]: Do we want to run this in auto generated files  (e.g. files created by Swagger)?

**Review by reivaj05 — APPROVED** [review]

**Review by kaankabalak — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]

File: verify-gofmt.sh (hunk 1 above)
Comment (cniackz) [reply verify-gofmt.sh:5]: @reivaj05 I would say yes as this sounds as a good practice for us to follow `gofmt` in all possible places. Let me try to expand this to all possible folders and see how it goes. Thanks for the feedback!

Comment (cniackz) [reply verify-gofmt.sh]: @reivaj05 expanded to all folder 👍

**Review by reivaj05 — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Comment by bexsoft** [comment]
Seems like we will have issues with gofmt on 1.19 every time we regenerate swagger files.

Not sure if should add an option during file regeneration to handle this.

**Comment by cniackz** [comment]
Sure @bexsoft that sounds like a good idea to me, I can add that format after generation of swagger files. Let me spend some time to add this on a separate PR.

**Comment by cniackz** [comment]
Hey @bexsoft what do you think about it?: https://github.com/minio/console/pull/2283

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**